src/style.css

```diff
@@ -0,0 +1,30 @@
+* {
+  font-family: Arial, sans-serif;
+  font-size: 13px;
+}
+
+html {
+  margin: 0;
+}
+
+/*body {*/
+/*  margin-top: 0;*/
+/*}*/
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;
+  margin-top: 5px;
+}
+
+h2 {
+  font-size: 15px;
+  font-weight: normal;
+  margin: 1em 0;
+}
```

Comment:
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice

Reply:
Done